TLM/TLM/Manager/Impl/LaneConnectionManager.cs

```diff
@@ -60,6 +61,35 @@ public class LaneConnectionManager
                    && connectedLanes.Any(laneId => laneId == targetLaneId);
         }
 
+
+        /// <summary>
+        /// determines whether or not the input lane is heading toward a start node.
+        /// </summary>
+        /// <returns>true if heading toward and start node.</returns>
+        private bool isHeadStartNode(uint sourceLaneId) {
```

Comment:
Name style, should begin with a capital letter.
Also making name a bit longer is ok. `IsHeadingTowardsStartNode` something like this?

Reply:
private functions also need to start with uppercase?